services/workflows-service/src/workflow/workflow-runtime-data.repository.ts

```diff
@@ -108,6 +109,33 @@ export class WorkflowRuntimeDataRepository {
     return this.findById(id, {}, projectIds);
   }
 
+  async patchDocumentById(
```

Comment:
The naming is misleading since you are not patching the document you're replacing him (like with PUT)